orbit/cmd/orbit/orbit.go

```diff
@@ -644,7 +644,7 @@ func main() {
 			log.Info().Err(err).Msg("failed to remove temporary osquery db")
 		}
 
-		if oi.HardwareUUID != orbitHostInfo.HardwareUUID {
+		if runtime.GOOS == "darwin" && oi.HardwareUUID != orbitHostInfo.HardwareUUID {
```

Comment:
we should probably do this check earlier and only get `oi` if we're on `darwin`

Reply:
Okay